sqe-http-api/Startup.cs

```diff
@@ -58,6 +61,21 @@ public void ConfigureServices(IServiceCollection services)
 			services.AddTransient<IArtefactRepository, ArtefactRepository>();
 			services.AddTransient<IDatabaseWriter, DatabaseWriter>();
 			services.AddTransient<ITextRepository, TextRepository>();
+			services.AddTransient<IRoiRepository, RoiRepository>();
+
+			services.AddResponseCompression();
+			services.Configure<BrotliCompressionProviderOptions>(options =>
+			{
+				// A custom compression level makes a huge difference CompressionLevel.Optimal uses level 11,
+				// which is incredibly slow.  A level between 5–7 gives a similarly sized result at a considerably
+				// faster speed. On one test Broti (CompressionLevel)5 compressed a 9.4 MB file to 1.57 MB, gzip
+				// compressed it to 2.45 MB (albeit a little bit faster).
+				options.Level = (CompressionLevel)5;
+			});
+			services.Configure<GzipCompressionProviderOptions>(options =>
+			{
+				options.Level = CompressionLevel.Optimal;
```

Comment:
This may cause stress on the server (not in development, though). Something to keep in mind when we stress test.

Reply:
Yeah, I should add a note about that and also that if we run this on something other than Kestrel (like IIS), then we should perhaps turn it off in favor what functionality that server provides (at least that is what the docs say).

Reply:
In that case, why don't you just remove it entirely for now? We're running the server and client on the same machine anyway (even if it's inside a docker container). Network bandwidth is meaningless. Compress only on an actual server.